wicketd/src/artifacts.rs

```diff
@@ -38,6 +40,7 @@ use crate::installinator_progress::IprArtifactServer;
 struct ArtifactsWithPlan {
     // TODO: replace with BufList once it supports Read via a cursor (required
     // for host tarball extraction)
+    system_version: Option<SemverVersion>,
```

Comment:
Nit - should this live inside `UpdatePlan`? I think they're always either `Some(_)` or `None` (never one of each), and it seems reasonable to say "the system version of the update plan we'll apply if you tell us to".

Reply:
Yeah, makes sense. Done.